src/lib/testing/create-test-env.ts

```diff
@@ -58,7 +57,7 @@ export function createTestEnvironment<State>(mocks: Mocks<State>, asserts: Asser
         }
 
         try {
-          throw new MissingTakeError(`Missing take assertion for action ${actionCreator.type}`);
+          throw new Error(`Missing take assertion for action ${actionCreator.type}`);
```

Comment:
@HenriBeck What was wrong with specific error types?

In OMS I just had a line where we had `e.message.includes('timeout')` where I would've been happier if I was able to do `e instanceof TimeoutError`.

Though maybe in the test env these are a little too specific and no-one uses their types. (Unless one is testing the test code?)

Reply:
I think here we don't need to have specific errors because I couldn't think of a reason to catch them and check if an error is a missing take error in the tests.

I think specific errors make sense in the actual source code of the library, so SagaCancelledError and the TimeoutError.

Reply:
Agreed!